api/src/main/java/org/apache/iceberg/PartitionSpec.java

```diff
@@ -450,13 +459,21 @@ Builder identity(String sourceName, String targetName) {
       return this;
     }
 
+    Builder identity(String sourceName, String targetName) {
+      return identity(findSourceColumn(sourceName), targetName);
+    }
```

Comment:
Nit: for the other cases, the methods are in a different order. That order also keeps the method signature line unchanged. I'd just move this above the `private` method.

Reply:
fixed